### ISSUE TITLE
Cannot dispatch mousedown event on nonexistent selector: xpath selector: //*[text()="Login"]

### ISSUE THREAD

**Issue by diegopetrucci** [clekstro/fedora_course_scraper#2]
I got this error running the script. Any idea why?
## 

Visiting site
CasperError: Cannot dispatch mousedown event on nonexistent selector: xpath selector: //*[text()="Login"]
  /usr/local/lib/node_modules/casperjs/modules/casper.js:1355 in mouseEvent
  /usr/local/lib/node_modules/casperjs/modules/casper.js:462 in click
  /Users/diegopetrucci/Desktop/fedora_course_scraper-master/course_scraper.coffee:49
  /usr/local/lib/node_modules/casperjs/modules/casper.js:1553 in runStep
  /usr/local/lib/node_modules/casperjs/modules/casper.js:399 in checkStep

**Comment by clekstro** [comment]
It's possible that the markup has changed, or that your initial URL is incorrect. Can you paste the course URL?

**Comment by diegopetrucci** [comment]
Of course! And, obviously, thanks for making this script and helping me.

URL: http://bitfountain.io/courses/complete-ios8
Title: The Bitfountain iOS 8 with Swift Immersive
